src/renderer/captureTool/js/capture.js

```diff
@@ -644,11 +642,13 @@
                 that.applier.change("prefsSetName", that.model.selectedPrefsSetName);
             }
             // that.render("5_confirmation");
+            that.applier.change("currentlySaving", true);
             that.saveCapturedPreferences();
         }
         else if (currentPage === "4_save_name") {
             // that.applier.change("currentPage", "5_confirmation");
```

Comment:
Probably a good opportunity to clean up these comments or at least explain why they are here

Reply:
Swept up!